src/fd.h

```diff
@@ -11,10 +11,16 @@
 #ifndef SQUID_FD_H_
 #define SQUID_FD_H_
 
+/// what kind of operation to count traffic for in fd_bytes()
```

Comment:
We should also make it possible to reuse these values in other, similar contexts, without changing this description:

```suggestion
/// distinguishes reading/importing I/O operations from their writing/exporting counterparts
```

I do not insist on any specific wording.

Reply:
I don't have any preference, rewording as suggested